testscript/doc.go

```diff
@@ -122,102 +122,107 @@ Additional conditions can be added by passing a function to Params.Condition.
+  - cd dir
+    Change to the given directory for future commands.
+
+  - chmod perm path...
+    Change the permissions of the files or directories named by the path arguments
+    to the given octal mode (000 to 777).
+
+  - [!] cmp file1 file2
+    Check that the named files have (or do not have) the same content.
+    By convention, file1 is the actual data and file2 the expected data.
+    File1 can be "stdout" or "stderr" to use the standard output or standard error
+    from the most recent exec or wait command.
+    (If the files have differing content and the command is not negated,
+    the failure prints a diff.)
+
+  - [!] cmpenv file1 file2
+    Like cmp, but environment variables in file2 are substituted before the
+    comparison. For example, $GOOS is replaced by the target GOOS.
+
+  - cp src... dst
+    Copy the listed files to the target file or existing directory.
+    src can include "stdout" or "stderr" to use the standard output or standard error
+    from the most recent exec or go command.
+
+  - env [key=value...]
+    With no arguments, print the environment (useful for debugging).
+    Otherwise add the listed key=value pairs to the environment.
+
+  - [!] exec program [args...] [&]
+    Run the given executable program with the arguments.
+    It must (or must not) succeed.
+    Note that 'exec' does not terminate the script (unlike in Unix shells).
+
+    If the last token is '&', the program executes in the background. The standard
+    output and standard error of the previous command is cleared, but the output
+    of the background process is buffered — and checking of its exit status is
+    delayed — until the next call to 'wait', 'skip', or 'stop' or the end of the
+    test. At the end of the test, any remaining background processes are
+    terminated using os.Interrupt (if supported) or os.Kill.
+
+    If the last token is '&word&` (where "word" is alphanumeric), the
+    command runs in the background but has a name, and can be waited
+    for specifically by passing the word to 'wait'.
+
+    Standard input can be provided using the stdin command; this will be
+    cleared after exec has been called.
+
+  - [!] exists [-readonly] file...
+    Each of the listed files or directories must (or must not) exist.
+    If -readonly is given, the files or directories must be unwritable.
+
+  - [!] grep [-count=N] pattern file
+    The file's content must (or must not) match the regular expression pattern.
+    For positive matches, -count=N specifies an exact number of matches to require.
+
+  - mkdir path...
+    Create the listed directories, if they do not already exists.
+
+  - unquote file...
+    Rewrite each file by replacing any leading ">" characters from
+    each line. This enables a file to contain substrings that look like
+    txtar file markers.
+    See also https://godoc.org/github.com/rogpeppe/go-internal/txtar#Unquote
+
+  - mv path1 path2
```

Comment:
Looks like both unquote and mv are out of alphabetical order here (yeah, unquote is an old problem, probably my fault, but worth fixing now too, i reckon).

Reply:
hadn't noticed we were trying to keep them sorted :) fixed in https://github.com/rogpeppe/go-internal/compare/24c64e725dba574088acf1dc892bf5fae970160b..2310f38754e627b9db74eba3c5327b73666b4576.